docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -160,6 +160,39 @@ airlock_microgateway:
           level: info
 ----
 
+== `gateways`
+
+[horizontal]
+type:: object
+default:: `{}`
+
+This parameter allows users to define `Gateway` resources to deploy.
+The component currently doesn't validate that the value of `spec.gatewayClassName` is a gateway class that's managed through parameter `gateway_classes`.
+
+=== Example
+
+[source,yaml]
+----
+airlock_microgateway:
+  gateways:
+    airlock/gateway-1:
+      spec:
+        gatewayClassName: airlock-microgateway
```

Comment:
```suggestion
        gatewayClassName: airlock-microgateway
        infrastructure:
          parametersRef:
            group: microgateway.airlock.com
            kind: GatewayParameters
            name: gatewayparameters
```
According the Microgateway [documentation](https://docs.airlock.com/microgateway/latest/index/1726159368876.html#CR_GatewayParameters) a reference to the `GatewayParameter` resource can be configured and exclusively overrides the reference to the `GatewayParameter` comming from the `GatewayClass`.

This also would give us the liberty having one `GatewayClass` for several `Gateways`. Not sure what's best here yet.

I think this behavior should be mentioned here.